numpy/random/_generator.pyi

```diff
@@ -0,0 +1,245 @@
+        dfden: _ArrayLikeFloat_co,
+        nonc: _ArrayLikeFloat_co,
+        size: Union[None, _ShapeLike] = ...,
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def chisquare(
+        self, df: _ArrayLikeFloat_co, size: Union[None, _ShapeLike] = ...
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def noncentral_chisquare(
+        self, df: _ArrayLikeFloat_co, nonc: _ArrayLikeFloat_co, size: Union[None, _ShapeLike] = ...
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def standard_t(
+        self, df: _ArrayLikeFloat_co, size: Union[None, _ShapeLike] = ...
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def vonmises(
+        self, mu: _ArrayLikeFloat_co, kappa: _ArrayLikeFloat_co, size: Union[None, _ShapeLike] = ...
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def pareto(
+        self, a: _ArrayLikeFloat_co, size: Union[None, _ShapeLike] = ...
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def weibull(
+        self, a: _ArrayLikeFloat_co, size: Union[None, _ShapeLike] = ...
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def power(
+        self, a: _ArrayLikeFloat_co, size: Union[None, _ShapeLike] = ...
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def laplace(
+        self,
+        loc: _ArrayLikeFloat_co = ...,
+        scale: _ArrayLikeFloat_co = ...,
+        size: Union[None, _ShapeLike] = ...,
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def gumbel(
+        self,
+        loc: _ArrayLikeFloat_co = ...,
+        scale: _ArrayLikeFloat_co = ...,
+        size: Union[None, _ShapeLike] = ...,
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def logistic(
+        self,
+        loc: _ArrayLikeFloat_co = ...,
+        scale: _ArrayLikeFloat_co = ...,
+        size: Union[None, _ShapeLike] = ...,
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def lognormal(
+        self,
+        mean: _ArrayLikeFloat_co = ...,
+        sigma: _ArrayLikeFloat_co = ...,
+        size: Union[None, _ShapeLike] = ...,
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def rayleigh(
+        self, scale: _ArrayLikeFloat_co = ..., size: Union[None, _ShapeLike] = ...
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def wald(
+        self,
+        mean: _ArrayLikeFloat_co,
+        scale: _ArrayLikeFloat_co,
+        size: Union[None, _ShapeLike] = ...,
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    def triangular(
+        self,
+        left: _ArrayLikeFloat_co,
+        mode: _ArrayLikeFloat_co,
+        right: _ArrayLikeFloat_co,
+        size: Union[None, _ShapeLike] = ...,
+    ) -> Union[float, ndarray[Any, dtype[float64]]]: ...
+    # Complicated, discrete distributions:
+    def binomial(
+        self, n: _ArrayLikeInt_co, p: _ArrayLikeFloat_co, size: Union[None, _ShapeLike] = ...
+    ) -> Union[int, ndarray[Any, dtype[int64]]]: ...
```

Comment:
```suggestion
    ) -> Union[int, ndarray[Any, dtype[int_]]]: ...
```
Same for the functions below:
`np.int_` always has the same size as `builtins.int`, which is platform-dependent (32-bit on windows; 64-bit on most other platforms).

Reply:
In `Generator` we freed ourselves from platform-specific C int sizes, so it either returns an array of `int64` or a Python `int`.

Reply:
Ah, I wasn't aware of that. 
Nevermind the previous comment in that case.